docs/data_management/lts/tutorial/individual_lts_tutorial.md

```diff
@@ -0,0 +1,113 @@
+
+Open a terminal using one of the interactive apps on Cheaha. Activate your conda environment created in the [Install s3cmd using within Anaconda Environment](./individual_lts_tutorial.md/#install-s3cmd-within-anaconda-environment-on-cheaha) section, and then run the below command:
+
+```bash
+s3cmd --configure
+```
+
+This will prompt you to enter the access key and secret key associated with your individual LTS account. You will be asked for additional information, which will be displayed on the screen, as shown below. You can copy the necessary details from the example provided [here](../interfaces.md/#configuring-s3cmd).
+
+![image-s3cmd](../images/config-s3cmd.png)
+
+Once the configuration is complete, `s3cmd` will generate a `.s3cfg` file in your home directory (`$HOME`), as shown below. To find your home directory in Cheaha and view the `.s3cfg` file, follow the instructions on our [Navigating Open OnDemand](../../../cheaha/open_ondemand/ood_layout.md/#navigating-open-ondemand) page. Be sure to check the **Show Dotfiles** option in the top right corner to make hidden files visible.
+
+![config-file](../images/s3cfg.png)
+
+### Creating Buckets
+
+Long Term Storage (LTS) services like Amazon S3 uses a specific data organization model based on **buckets** and **objects**. Think of buckets as folders that contain individual pieces of data called objects. Each bucket has a unique name. We have documentation about basic terminology on s3 storage system [here](../index.md/#terminology) and a naming guide for LTS buckets [here](../index.md/#avoiding-duplicate-names-for-buckets).
+
+Once you have complete `s3cmd` configuration, you can create new buckets in your individual LTS storage. To create a bucket use a `mb`(make bucket) command:
+
+```bash
+s3cmd mb s3://your_bucket_name
+```
+
+Please replace `your_bucket_name` with your desired name. This command creates a new bucket that you named `your_bucket_name` in your LTS storage using the currently configured `s3cmd` profile.
+
+The image below shows the LTS bucket created successfully after running the command.
+
+![image-bucket](../images/create-bucket.png)
+
+### Managing Buckets
+
+To manage a bucket, various commands can be used. Below are some common `s3cmd` commands to interact with your LTS bucket and its objects:
+
+- To **lists all buckets** accessible with your current `s3cmd` profile, use command: `s3cmd ls`.
+- To **lists all files in a bucket** accessible with your current `s3cmd` profile, use command: `s3cmd ls s3://your_bucket_name`.
+- To **upload a file**  named `file.txt` to a bucket, use a command: `s3cmd put file.txt s3://your_bucket_name/`.
+- To **download a file** a file named `file.txt` from a bucket, use a command:`s3cmd get s3://your_bucket_name/file.txt`.
```

Comment:
Show an additional example of how to download to a specific full path. I don't know the flags for this, but surely it can `get` to a directory of your choosing.